alex-garcia/lesson-1/homework-2-car-project/CarProject/src/CarBehavior.java

```diff
@@ -0,0 +1,126 @@
+//https://vehiculos.doncomos.com/calcular-consumo-gasolina
+
+class CarBehavior {
+    static class Car {
+        String name;
+        double fuel_capacity;
+        double fuel;
+        double maximum_distance_with_full_tank;
+        double distance;
+        boolean status;
+    }
+
+    //Fields:
+    private  Car [] automobile;
+    private int automobileCount;
+
+    //Constructor;
+    CarBehavior(){
+        automobile = new Car[3];
+        automobileCount = 0;
+    }
+
+    // Methods:
+    void addCar(Car Car){
```

Comment:
Same here, start with lowercase for Fields, Methods and Parameters

Reply:
Update: Parameters started with lower case